news/models.py

```diff
@@ -70,8 +70,10 @@ def label_from_instance(self, obj):
 
 class NewsItemAdminForm(forms.ModelForm):
     create_user = CustomModelChoiceField(
-        queryset=get_user_model().objects.filter(is_active=True).order_by(
-            'last_name', 'first_name'
+        queryset=get_user_model().objects.filter(is_active=True).exclude(
+            first_name='',
+            last_name='',
+        ).order_by('last_name', 'first_name')
         ).all()
```

Comment:
I think you need to drop this line. .all() is implied when filtering, and the closing paren before it is extra.
```suggestion
```

Reply:
Gah, of course. I also need to move `order_by` back in front, per @willbarton's advice on performance.

Reply:
Updated.